src/Form/Form.tsx

```diff
@@ -229,6 +238,15 @@ const FormSection = <
       {title && <Typography variant={"h6"}>{title}</Typography>}
       {properties.map(property => {
         let input: React.ReactNode = null;
+        const inputValidationErrors = validationErrors?.[property.name];
```

Comment:
is it fine if property.name does not exist ?

Reply:
Sure !

Reply:
`properties` items have the `name` attribute as required, so `property.name` should always be defined